### PR TITLE
Background status

### PR DESCRIPTION
**Pull request by GeorgesStavracas**
This is the libportal counterpart to https://github.com/flatpak/xdg-desktop-portal/pull/901

Made with love

### CODE REVIEW

File: libportal/background.c [hunk 1]
```diff
@@ -20,9 +20,115 @@
...
+  GTask *task;
+  char *status_message;
+} SetStatusCall;
+
+static void
+set_status_call_free (SetStatusCall *call)
+{
+  g_clear_pointer (&call->status_message, g_free);
+  g_clear_object (&call->portal);
+  g_clear_object (&call->task);
```
Comment (jadahl) [inline libportal/background.c:38]: Should `g_free (call);` too shouldn't it?

Comment (GeorgesStavracas) [reply libportal/background.c:38]: Correct, my bad

**Review by jadahl — APPROVED** [review]
lgtm now with the leak fixed.

**Comment by GeorgesStavracas** [comment]
Thanks!

**Comment by matthiasclasen** [comment]
This looks fine to me. Thanks for doing that work!

**Pull request merged**